jbmc/regression/jbmc/NondetEnumOpaqueReturn/test_indirect_enum_initalization.desc

```diff
@@ -0,0 +1,5 @@
+CORE
+indirect/IndirectVirtualCall.class
+--function indirect.IndirectVirtualCall.main
+--
+^Invariant check failed$
```

Comment:
Tests should almost always check for EXIT=0 and SIGNAL=0 -- otherwise this negative assertion is easily satisfied by e.g. immediately aborting the process